roofit/roofitcore/src/Initialisation.cxx

```diff
@@ -0,0 +1,100 @@
+#include "RConfigure.h"
+#include "TEnv.h"
+#include "TSystem.h"
+
+#include <iostream>
+#include <algorithm>
+#include <vector>
+#include <string>
+#include <exception>
+
+/**
+\file Initialisation.cxx
+\ingroup Roofitcore
+
+Run static initialisers on first load of RooFitCore.
+**/
+
+namespace {
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+/// Print RooFit banner.
+void doBanner() {
+#ifndef __ROOFIT_NOBANNER
+  if (gEnv->GetValue("RooFit.Banner", 1) == 0)
+    return;
+
+  /// RooFit version tag.
+  constexpr char VTAG[] = "3.60";
```

Comment:
I guess it is better to specify this through the build system and maybe a `-D__ROOFIT_VTAG=...` compiler option.

Reply:
I'm not sure that's necessary. This tag didn't change since 2011, and I don't see that it's happening soon. I didn't want to interfere with the work of the original authors, though, so I left it in place.

Reply:
I think it's a good practice to separate these constants from the code; however, given that it has been inherited from the original authors, we can leave it in place and fix it later if need be.